src/modules/SdnDiag.Health.psm1

```diff
@@ -2373,6 +2373,46 @@ function Test-SdnHostAgentConnectionStateToApiService {
     return $sdnHealthTest
 }
 
+function Test-SdnVfpEnabledVMSwitch {
+    <#
+        .SYNOPSIS
+            Enumerates the VMSwitches on the system and validates that only one VMSwitch is configured with VFP.
+    #>
+
+    [CmdletBinding()]
+    param()
+
+    Confirm-IsServer
+    $sdnHealthTest = New-SdnHealthTest
+    $i = 0
+
+    try {
+        $vmSwitches = Get-VMSwitch
```

Comment:
Can we make this a generic utility method and invoke that to see the results ? This way the fucntionality can be accessed more broadly.

Reply:
ANd we could expose that into the psd1 too

Reply:
@sbgms can you clarify more? Are you wanting something that returns back a PSObject?